extensions/commands/art/cmd_build_info.py

```diff
@@ -319,11 +321,14 @@ def _check_min_required_conan_version(min_ver):
                              f"Conan versions>={min_ver}. Please update Conan.")
 
 
-def _add_default_arguments(subparser, is_bi_create=False):
+def _add_default_arguments(subparser, is_bi_create=False, is_bi_create_bundle=False):
     url_help = "Artifactory url, like: https://<address>/artifactory."
     if is_bi_create:
         url_help += " This may be not necessary if all the information for the Conan artifacts is present in the " \
                     "local cache."
+    if not (is_bi_create_bundle or is_bi_create_bundle):
```

Comment:
```suggestion
    if not is_bi_create_bundle:
```

Reply:
The `--project` flag does not apply to the bi-create neither to bi-create-bundle, so we have to keep that